appinventor/codelabs/paint-pot/index.html

```diff
@@ -0,0 +1,221 @@
+      <p>Your app should look like this:</p>
+      <img src="img/part1_ui.png">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add button event handlers">
+      <p>Now it's time to code our app. Go to <b>Blocks View</b>.</p>
+      <p>Open the <b>ButtonRed</b> drawer and drag out a <code>when ButtonRed.Click</code> block.</p>
+      <p>Open the <b>DrawingCanvas</b> drawer and drag out a <code>set DrawingCanvas.BackgroundColor to</code> block. From the drop down menu, change <code>BackgroundColor</code> to <code>PaintColor</code>. Place this block inside <code>when ButtonRed.Click</code>.</p>
+      <p>Open the <b>Colors</b> drawer and drag out the block for the color Red. Place it into the <code>set DrawingCanvas.PaintColor to</code> block.</p>
+      <p>Repeat these steps for the Blue and Green buttons.</p>
+      <p>For the wipe button, drag a <code>when ButtonWipe.Click</code> block into the workspace and place a <code>call DrawingCanvas.Clear</code> block into it.</p>
+      <p>Your buttons blocks should look like this:</p>
+      <img src="img/button_blocks.png" style="max-width: 400px">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add touch event handlers">
+      <p>Open the <b>DrawingCanvas</b> drawer and drag a <code>when DrawingCanvas.Touched</code> block into the workspace. From the <b>DrawingCanvas</b> drawer drag out a <code>call DrawingCanvas.DrawCircle</code> block and place it inside the <code>when DrawingCanvas.Touched</code> block.</p>
+      <p>To fill in the rest of the block, move your cursor over the x variable. Get the <code>get x</code> block and place it in the <code>centerX</code> socket. Do the same for y.</p>
+      <p>For <code>radius</code>, drag a blank number block from the <b>Math</b> drawer, and type 5. For the <code>fill</code> socket, it should say <code>true.</code>
+      <p>Your final <code>when DrawingCanvas.Touched</code> block should look like this:</p>
+      <img src="img/touch_block.png" style="max-width: 450px">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add drag events">
+      <p>From the <b>DrawingCanvas</b> drawer, drag a <code>when DrawingCanvas.Dragged</code> block into the workspace, and drag a <code>call DrawingCanvas.DrawLine</code> block into the block.</p>
+      <h2>Challenges</h2>
+        <ol>
+          <li>Fill in the rest of the block like this:</li>
+          <img src="img/dragged_block.png" style="max-width: 450px">
+        </ol>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Checkoff #1">
+      <aside class="special">
+          <h2>paint-pot-1</h2>
+          <p><b>To receive this checkoff, you must</b></p>
+          <ul class="checklist">
+            <li>Demonstrate your working app</li>
+            <li>Explain what local variables are (the orange arguments built into some event handlers)</li>
```

Comment:
I would remove these two bullets that talk about the local variables. We don't mention it anywhere.